peachjam/migrations/0048_backfill_extracted_citations.py

```diff
@@ -0,0 +1,21 @@
+# Generated by Django 3.2.17 on 2023-02-09 13:53
+
+from django.db import migrations
+
+
+def backfill_extracted_citations(apps, schema_editor):
+    CoreDocument = apps.get_model("peachjam", "CoreDocument")
+    for doc in CoreDocument.objects.all().iterator(chunk_size=100):
+        Work = apps.get_model("peachjam", "Work")
+        work = Work.objects.get(frbr_uri=doc.work_frbr_uri)
+        work.update_extracted_citations()
```

Comment:
When accessing the model like this in a migration, the model only has the basic structure. It doesn't have any custom code associated with it. So this won't work, because `update_extracted_citations` won't exist on that class.

We only need to backfill once-off for all our sites. There are a few options

1. do this manually outside of the code by running the above snippet of code manually outside of the deployment.
2. add a management command (which we only ever need once, so it's overkill)
3. duplicate the business logic here directly -- this is the simplest from an operational perspective and what I recommend